packages/web-components/src/components/ic-page-header/ic-page-header.tsx

```diff
@@ -180,12 +180,14 @@ export class PageHeader {
           ["sticky"]: sticky,
           ["sticky-desktop"]: !sticky && stickyDesktopOnly,
         }}
+        aria-label="page header"
       >
         <header
```

Comment:
If we're stripping <header>'s semantic value with the role=presentation, is there any reason to keep it as a <header> element?  Could we replace it with a <div> (and therefore not need the role=presentation)

Reply:
I think changing the element could make it a breaking change if users are looking for `<header>` in tests, but if we think this is the best way around it then I could move this into the v3 work

Reply:
darn, that is tricky. I don't want to prevent this fix going into V2. Would two PRs (one here for the base fix, then once this is merged in another for that refactor) be too much?

Reply:
Just been having a look around and I think it should still stay as a `<header>` element, it looks like under the hood it does more than setting the role to banner

Reply:
Righto, no worries then :)